services/filemanager/azureBlobStoragemanager.go

```diff
@@ -131,6 +135,8 @@ func (manager *AzureBlobStorageManager) ListFilesWithPrefix(prefix string, maxIt
 	}
 
 	// List the blobs in the container
+	ctx := context.Background()
+	marker := string("")
```

Comment:
Generally more common to do:

```suggestion
    var marker string
```
When you can provide a useful value for something.